qkernel/src/interrupt/aarch64/mod.rs

```diff
@@ -1,6 +1,137 @@
+use super::super::syscall_dispatch_aarch64;
 pub unsafe fn InitSingleton() {
 }
 
 pub fn init() {
-    // TODO load IDT
-}
\ No newline at end of file
+    // TODO set up GIC
+}
+
+use core::arch::asm;
+
+#[repr(C)]
+pub struct ExceptionStateEL1{
+    pub spsr_el1: u64,
+    pub elr_el1: u64,
+    pub esr_el1: u64,
+    pub tpidr_1l0: u64,
```

Comment:
"_1l0" -> maybe typo here?

Reply:
yes it is, I'll fix it in the next revision.